x-pack/solutions/chat/plugins/workchat-app/server/saved_objects/agents.ts

```diff
@@ -15,7 +15,7 @@ export const agentSoType: SavedObjectsType<AgentAttributes> = {
   hidden: true,
   namespaceType: 'agnostic',
   mappings: {
-    dynamic: 'strict',
```

Comment:
This will cause error, you need to restart local es from fresh state (or without internal index)